src/main.js

```diff
@@ -123,95 +124,177 @@ var quotes = [
   "Each person must live their life as a model for others.",
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
+
 var savedPosters = [];
-var currentPoster = {};
 
-savedPostersPage.style.display = "none"
-posterForm.style.display = "none"
+var currentPoster = {};
```

Comment:
Was this one of the variables that was already defined for us or did we add this one? I know savedPosters was given to us but I don't remember if this one was as well.